documentation/modules/collected-logs-cr-info.adoc

```diff
@@ -21,11 +21,11 @@ The `must-gather` tool collects the following logs and CR files in an archive fi
 * Logs:
 ** `importer` pod: Disk-to-data-volume conversion log.
 +
-The pod naming convention is `importer-<migration_plan>-<vm_id>-<5_char_id>` for VMware and `importer-<migration_plan>-<partial_vm_id><5_char_id>` for {rhv-short}, for example, `importer-mig-plan-ed90dfc6-9a17-4a8btnfh`, where `ed90dfc6-9a17-4a8` is the partial VM ID and `btnfh` is the 5-character ID.
+The pod naming convention is `importer-<migration_plan>-<vm_id>-<5_char_id>` for VMware and `importer-<migration_plan>-<partial_vm_id><5_char_id>` for {rhv-short}, for example, `importer-mig-plan-ed90dfc6-9a17-4a8btnfh`, where `ed90dfc6-9a17-4a8` is the partial VM ID and `btnfh` is the 5-character ID. If the combined migration plan name and VM ID are very long, the pod name might be truncated to make room for the generated 5-character ID.
 
 ** `conversion` pod: VM conversion log. The `conversion` pod runs `virt-v2v`, which installs and configures device drivers on the PVCs of the VM.
 +
-The pod naming convention is `<migration_plan>-<vm_id>-<5_char_id>` for VMware and `<migration_plan>-<partial_vm_id><5_char_id>` for {rhv-short}.
+The pod naming convention is `<migration_plan>-<vm_id>-<5_char_id>` for VMware and `<migration_plan>-<partial_vm_id><5_char_id>` for {rhv-short}. If the combined migration plan name and VM ID are very long, the pod name might be truncated to make room for the generated 5-character ID.
 
```

Comment:
In that case, looks like the pod naming convention for VMware/RHV is the same, but for RHV the hyphen and part of VM id were removed to make place for the 5-character ID.
I think we don't need to separate the naming convention to vmware and rhv

Reply:
I agree. The hyphen is not important enough to deserve a separate sentence. I will combine them.